src/thunderbird_accounts/mail/views.py

```diff
@@ -83,26 +83,21 @@ def sign_up_submit(request: HttpRequest):
         pass
 
     user = request.user
+
+    # Set the user's username to the new primary email address and save the user
+    user.username = email_address
+    user.save()
```

Comment:
This route will probably be deprecated soon, but for the sake of completion I'm ensuring we're saving the brand new thundermail address as the user's username.